src/app/shared/picker/color-picker/color-picker.component.html

```diff
@@ -1,16 +1,20 @@
-<div class="rounded" *ngFor="let colorOption of greyscaleColors" [style.background]="colorOption"
-     [style.border-color]="colorHighlight(colorOption)"
-     (mouseenter)="preview(colorOption)" (mouseleave)="preview()" (click)="select(colorOption)"></div>
-<div class="rounded control-icon {{!isCustom() ? 'mt-0 p-0' : ''}}"
-     [style.border-color]="isCustom() ? colorHighlight(selected) : 'transparent'"
-     [style.background]="isCustom() ? selected : 'transparent'"
-     id="spectrum-picker-{{id}}">
-  <i [style.color]="isCustom() ? 'transparent' : ''" class="far fa-crosshairs" (click)="openSpectrum()"></i>
+<div class="text-center text-muted px-0" i18n="@@color-picker.select.icon">1. Choose color</div>
+<div class="colors justify-content-center">
+  <div class="rounded" *ngFor="let colorOption of greyscaleColors" [style.background]="colorOption"
+       [style.border-color]="colorHighlight(colorOption)"
```

Comment:
Again. Using functions to bind template values is not a good idea. Try to use public properties or pipes instead.

Reply:
This was there all the time. I removed some of the functions already!!!

Reply:
Fixed